readme.md

```diff
@@ -59,13 +97,50 @@ $ cd test/LowerColorado_TX
 $ python3 -m nwm_routing -V3 -f test_AnA.yaml
```

Comment:
I think we can include two main test cases one for NHD and the other for HYfeature.  So, for example, -V4 -f test_AnA_V4_NHD.yaml and -V -f test_AnA_V4_HYFeature.yaml as V4 is the most update main function.

Reply:
Good point. I'll update that